src/main/java/com/cs3250p1/project1/BarChartEx.java

```diff
@@ -91,5 +101,44 @@ private void update(String date) {
         }
         
     }
+
+    private static class reportPanel extends JPanel{
```

Comment:
okay, report presentation is a nice addition, that should make our data visualization enough for now. Now we can move onto finishing touches and making things look presentable or something